sdcm/cluster.py

```diff
@@ -371,25 +371,25 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
-                                       DatabaseLogEvent(type='BAD_ALLOC', regex='std::bad_alloc'),
-                                       DatabaseLogEvent(type='SCHEMA_FAILURE', regex='Failed to load schema version'),
-                                       DatabaseLogEvent(type='RUNTIME_ERROR', regex='std::runtime_error'),
-                                       DatabaseLogEvent(type='FILESYSTEM_ERROR', regex='filesystem_error'),
-                                       DatabaseLogEvent(type='STACKTRACE', regex='stacktrace'),
-                                       DatabaseLogEvent(type='BACKTRACE', regex='backtrace', severity=Severity.ERROR),
-                                       DatabaseLogEvent(type='SEGMENTATION', regex='segmentation'),
-                                       DatabaseLogEvent(type='INTEGRITY_CHECK', regex='integrity check failed'),
-                                       DatabaseLogEvent(type='REACTOR_STALLED', regex='Reactor stalled'),
-                                       DatabaseLogEvent(type='SEMAPHORE_TIME_OUT', regex='semaphore_timed_out'),
-                                       DatabaseLogEvent(type='BOOT', regex='Starting Scylla Server',
-                                                        severity=Severity.NORMAL),
-                                       DatabaseLogEvent(type='SUPPRESSED_MESSAGES', regex='journal: Suppressed',
-                                                        severity=Severity.WARNING)]
+        self._database_error_events = [
+            DatabaseLogEvent(type='NO_SPACE_ERROR', regex='No space left on device'),
+            DatabaseLogEvent(type='UNKNOWN_VERB', regex='unknown verb exception', severity=Severity.WARNING),
+            DatabaseLogEvent(type='DATABASE_ERROR', regex='Exception '),
+            DatabaseLogEvent(type='BAD_ALLOC', regex='std::bad_alloc'),
+            DatabaseLogEvent(type='SCHEMA_FAILURE', regex='Failed to load schema version'),
+            DatabaseLogEvent(type='RUNTIME_ERROR', regex='std::runtime_error'),
+            DatabaseLogEvent(type='FILESYSTEM_ERROR', regex='filesystem_error'),
+            DatabaseLogEvent(type='STACKTRACE', regex='stacktrace'),
+            DatabaseLogEvent(type='BACKTRACE', regex='backtrace', severity=Severity.ERROR),
+            DatabaseLogEvent(type='SEGMENTATION', regex='segmentation'),
+            DatabaseLogEvent(type='INTEGRITY_CHECK', regex='integrity check failed'),
+            DatabaseLogEvent(type='REACTOR_STALLED', regex='Reactor stalled'),
+            DatabaseLogEvent(type='SEMAPHORE_TIME_OUT', regex='semaphore_timed_out'),
+            DatabaseLogEvent(type='BOOT', regex='Starting Scylla Server', severity=Severity.NORMAL),
+            DatabaseLogEvent(type='SUPPRESSED_MESSAGES', regex='journal: Suppressed', severity=Severity.WARNING),
+            DatabaseLogEvent(type='CQL_SERVER_CONN_SYSTEM_ERROR', severity=Severity.WARNING,
```

Comment:
You should put this new one at the top of the list, otherwise it be raised as DATABASE_ERROR